docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -1078,22 +1086,30 @@
       "\tDomain ID      =\t10000\n",
       "\tNuclide        =\tU235\n",
       "\tCross Sections [cm^-1]:\n",
-      "            Group 1 [0.625      - 20000000.0eV]:\t8.05e-03 +/- 3.37e-01%\n",
-      "            Group 2 [0.0        - 0.625     eV]:\t3.62e-01 +/- 6.02e-01%\n",
+      "            Group 1 [0.625      - 20000000.0eV]:\t8.05e-03 +/- 3.37e-01\n",
```

Comment:
Actually it is relative error. This was a bone-headed move by me.  It should have a % after it, but in my commit you're commenting on I removed the %.  For some reason I saw it and thought it was left over from converting from the old python string formatting method (which is `%`) to the newer `str.format()` method.  For some reason I didnt think it was actually percent :-( 

Its fixed now though